[DOCS] converting rst to md in the installation folder (#2595)
open-edge-platform/edge-ai-suites: robotics-ai-suite/docs/troubleshooting.md

   ['CPU']
   ```

>    Please firstly make sure you have installed the [firmware](embodied/get-started/installation/gpu_firmware.md) and [driver](embodied/get-started/installation/client_gpu_driver.md) for GPU. If you are still encountering this issue:

Reviewer: ```suggestion
   First, make sure you have installed the [firmware](embodied/get-started/installation/gpu_firmware.md) and [driver](embodied/get-started/installation/client_gpu_driver.md) for GPU. If you are still encountering this issue:
```

   RuntimeError: Native API failed. Native API returns: -1102 (PI_ERROR_UNINITIALIZED) -1102 (PI_ERROR_UNINITIALIZED)
   ```

>    To run IPEX workloads, please uninstall the NPU debs package installed as according to [NPU firmware](embodied/get-started/installation/npu_firmware.md).

Reviewer: ```suggestion
   To run IPEX workloads, please uninstall the NPU deb packages in the [NPU firmware](embodied/get-started/installation/npu_firmware.md).
```

   failed to unassign iommu device!
   ```

>    1. Please check VT-d Enabled in BIOS, refer to the BIOS configuration listed in [OS Setup](embodied/get-started/prerequisites/os_setup.md).

Reviewer: ```suggestion
   1. Check if VT-d is enabled in BIOS. Refer to the BIOS configuration listed in [OS Setup](embodied/get-started/prerequisites/os_setup.md).
```
